app/src/main/java/no/usn/mob3000/ui/components/settings/SettingsSectionApplication.kt

```diff
@@ -84,14 +86,14 @@ fun SettingsSectionApplication(
             onDismissRequest = { languageExpanded = false }
         ) {
             DropdownMenuItem(
-                text = { Text(stringResource(R.string.settings_section_app_language_enus)) },
+                text = { Text(stringResource(R.string.settings_section_app_language_enus),color = MaterialTheme.colorScheme.onBackground) },
                 onClick = {
                     onLanguageChange("English")
                     languageExpanded = false
                 }
             )
             DropdownMenuItem(
-                text = { Text(stringResource(R.string.settings_section_app_language_nonb)) },
+                text = { Text(stringResource(R.string.settings_section_app_language_nonb),color = MaterialTheme.colorScheme.onBackground) },
```

Comment:
This should be indented, since there are two parameters now.
```kotlin
DropdownMenuItem(
    text = {
        Text(
            stringResource(R.string.settings_section_app_language_nonb),
            color = MaterialTheme.colorScheme.onBackground)
    },
```